tests/backend/realm/test_realm_stats.py

```diff
@@ -0,0 +1,39 @@
+# Parsec Cloud (https://parsec.cloud) Copyright (c) AGPLv3 2019 Scille SAS
+
+import pytest
+from uuid import uuid4
+
+from tests.backend.common import realm_create, realm_stats
+from tests.backend.common import vlob_create, block_create
+
+
+@pytest.mark.trio
+async def test_realm_stats_ok(backend, alice, alice_backend_sock, administration_backend_sock, realm):
```

Comment:
`administration_backend_sock` fixture is not needed for this test
`backend` fixture is indirectly needed by `alice_backend_sock` so no need to declare it here as well

same thing for `alice`